index.Rmd

```diff
@@ -487,6 +488,58 @@ getdoge(c("wow", "suchhostility", "bemoreinclusive"))
 
 ***Now, back to your regular, cat-dominated tutorial.***
 
+## <a href="#dataNames" name="dataNames"/>#</a> Data from the web: names
+
+
+Besides pictures, R is also really good at scraping data from websites! Here, we'll look at the names of famous cats that are listed on Wikipedia. 
+
+First, we'll have to load some helpful packages.
+```{r scrapePackages, collapse=TRUE}
+library(XML)
+library(RCurl)
```

Comment:
Can you use `httr` instead?  It is more actively maintained, and is easier to use